app/models/accreditation_institute_campus.rb

```diff
@@ -6,8 +6,10 @@ class AccreditationInstituteCampus < ApplicationRecord
   include CsvHelper
 
   # rubocop:disable Rails/HasManyOrHasOneDependent
-  has_many :accreditation_records, primary_key: :dapip_id, foreign_key: 'dapip_id'
-  has_many :accreditation_actions, primary_key: :dapip_id, foreign_key: 'dapip_id'
+  has_many(:accreditation_records, primary_key: :dapip_id, foreign_key: 'dapip_id',
+                                   inverse_of: :accreditation_institute_campus)
+  has_many(:accreditation_actions, primary_key: :dapip_id, foreign_key: 'dapip_id',
+                                   inverse_of: :accreditation_institute_campus)
   # rubocop:enable Rails/HasManyOrHasOneDependent
```

Comment:
Did you attempt to remove the rule exemption here?

Reply:
I did not, should auditing existing `rubocop:disables` within repository be done?

Reply:
That might be a useful track of work depending on bandwidth for tech-debt, but not necessary for this PR. :+1: